src/Features/CSharp/Portable/GenerateMember/GenerateParameterizedMember/CSharpGenerateDeconstructMethodService.cs

```diff
@@ -35,5 +41,46 @@ protected override bool AreSpecialOptionsActive(SemanticModel semanticModel)
 
         protected override bool IsValidSymbol(ISymbol symbol, SemanticModel semanticModel)
             => CSharpCommonGenerationServiceMethods.IsValidSymbol();
+
+        public override ImmutableArray<IParameterSymbol> TryMakeParameters(SemanticModel semanticModel, SyntaxNode target, ISemanticFactsService semanticFacts, CancellationToken cancellationToken)
+        {
+            ITypeSymbol targetType;
+            if (target is PositionalPatternClauseSyntax positionalPattern)
```

Comment:
can you give an example here of what the original user code would look like and what you are generating?